runtime/extensions/CMakeLists.txt

```diff
@@ -30,5 +30,29 @@ if(ENABLE_LIGHTNING_KOKKOS)
 
     FetchContent_MakeAvailable(pennylane_lightning_kokkos)
     
-    target_link_libraries(lightning_interfaces INTERFACE pennylane_lightning_kokkos)
+    target_link_libraries(rt_interfaces INTERFACE pennylane_lightning_kokkos)
```

Comment:
@maliasadi do the installation instructions need updating?

Reply:
The installation instructions remain the same as they're relying on Make commands.